### PR TITLE
[YUNIKORN-953] Investigate of kubeflow/training-operator

### PR DESCRIPTION
**Pull request by lowc1012**
Update the run_tensorflow.md

### CODE REVIEW

File: docs/user_guide/workloads/run_tensorflow.md [hunk 1]
```diff
@@ -24,17 +25,69 @@ specific language governing permissions and limitations
...
-* CRD: https://github.com/kubeflow/manifests/tree/master/tf-training/tf-job-crds/base
-* Deployment: https://github.com/kubeflow/manifests/tree/master/tf-training/tf-job-operator/base
-Also you can install kubeflow which can auto install tf-operator for you, URL: https://www.kubeflow.org/docs/started/getting-started/
+This guide gives an overview of how to set up [training-operator](https://github.com/kubeflow/training-operator)
+and how to run a Tensorflow job with YuniKorn scheduler. The training-operator is a unified training operator maintained by
+Kubeflow. It not only supports TensorFlow but also PyTorch, XGboots, etc.
 
-A simple Tensorflow job example:
+## Install training-operator
+You can run one command to install training operator in kubeflow namespace. If you have problems with installation,
```
Comment (yangwwei) [inline docs/user_guide/workloads/run_tensorflow.md:33]: You can the following command to install the training operator in the `kubeflow` namespace, please refer to [this doc|xxx] for details.

File: docs/user_guide/workloads/run_tensorflow.md [hunk 2]
```diff
@@ -24,17 +25,69 @@ specific language governing permissions and limitations
...
+please refer to [this doc](https://github.com/kubeflow/training-operator#installation) for more details.
+```
+kubectl apply -k "github.com/kubeflow/training-operator/manifests/overlays/standalone?ref=v1.3.0"
```
Comment (yangwwei) [inline docs/user_guide/workloads/run_tensorflow.md:36]: missing `-n kubeflow` in this command? since earlier it said to install in kubeflow namespace

**Review by yangwwei — CHANGES_REQUESTED** [review]
hi @lowc1012  thanks for working on this. Overall looks good, pls see some minor comments.

File: docs/user_guide/workloads/run_tensorflow.md (hunk 2 above)
Comment (lowc1012) [reply docs/user_guide/workloads/run_tensorflow.md:36]: Thanks @yangwwei 
The manifests is in kubeflow namespace by default, so "-n kubeflow" can be omitted.

ref: https://github.com/kubeflow/training-operator/blob/master/manifests/overlays/standalone/kustomization.yaml#L3

Comment (yangwwei) [reply docs/user_guide/workloads/run_tensorflow.md:36]: Ah, I see

**Review by yangwwei — APPROVED** [review]
LGTM, thanks for getting this documented

**Pull request merged**